manim/mobject/matrix.py

```diff
@@ -214,18 +338,76 @@ def __init__(
 
 
 class IntegerMatrix(Matrix):
+    """A mobject that displays a matrix with integer entries on the screen."""
+
     def __init__(self, matrix, element_to_mobject=Integer, **kwargs):
+        """
+        Note- Will round if there are decimal entries in the matrix.
+
+        Parameters
+        ----------
+        matrix : typing.Iterable
+            A numpy 2d array or list of lists
+        element_to_mobject : Mobject, optional
+            Mobject to use, by default Integer
```

Comment:
```suggestion
        matrix : :class:`typing.Iterable`
            A numpy 2d array or list of lists
        element_to_mobject : :class:`Mobject`, optional
            Mobject to use, by default Integer
```

Reply:
Have incorporated all the suggestions. Thanks.